internal/receiver/prometheusremotewritereceiver/README.md

```diff
@@ -30,4 +30,19 @@ If you're using the [native remote write configuration](https://prometheus.io/do
 If possible, wait on sending multiple requests until you're reasonably assured that metadata has propagated to the receiver.
 
 ## Nuances in translation
-We do not [remove suffixes](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6658646e7705b74f13031c777fcd8dd1cd64c850/receiver/prometheusreceiver/internal/metricfamily.go#L316) as is done in the otel-contrib `prometheusreceiver`
+- We do not [remove suffixes](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6658646e7705b74f13031c777fcd8dd1cd64c850/receiver/prometheusreceiver/internal/metricfamily.go#L316) as is done in the otel-contrib `prometheusreceiver`
+- Keep in mind promethes timestamps are in unix epoch milliseconds, while otel timestamps are in unix epoch nanoseconds
```

Comment:
```suggestion
- Keep in mind prometheus timestamps are in unix epoch milliseconds, while otel timestamps are in unix epoch nanoseconds
```

What does that mean, should we expect the precision of timestamps to go to milliseconds? Why are we calling that out?